packages/jaeger-ui/src/components/TracePage/TraceStatistics/index.tsx

```diff
@@ -42,75 +39,6 @@ type State = {
   valueNameSelector2: string | null;
 };
 
-const columnsArray: any[] = [
```

Comment:
why did we delete this structure? It looks like it contained a lot of semantic details that you could've used to construct the columns automatically, like using `suffix` to decide which render should be used, or `isDecimal` to decide which comparison function to use. It would also make the PR smaller.

Reply:
Yep modified in that way